CMakeLists.txt

```diff
@@ -157,16 +157,13 @@ write_basic_package_version_file(eudaqConfigVersion.cmake
 # Install the export set for use with the install-tree
 install(EXPORT eudaqTargets
   NAMESPACE eudaq::
-  DESTINATION ${PROJECT_BINARY_DIR}
+  DESTINATION cmake
   COMPONENT dev)
 
-file(GLOB TARGET_FILES "${PROJECT_BINARY_DIR}/eudaqTarget*.cmake")
```

Comment:
main change here.   eudaqTargets goes into ${PROJECT_BINARY_DIR}  which is 'build' folder, instead of a installed folder